From 0264bc3c8044c83a35a1a487e84483db53d49c8c Mon Sep 17 00:00:00 2001
From: Chen Jun <chenjun102@huawei.com>
Date: Tue, 23 Mar 2021 09:05:18 +0000
Subject: [PATCH] powerpc: Do not compile any dts if CONFIG_OF_ALL_DTBS=y
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 0264bc3c8044c83a35a1a487e84483db53d49c8c


hulk inclusion
category:bugfix
bugzilla:50795
CVE:NA

-------------------

There are many warning when make the dts of powerpc.
And those dts would not be used.

Signed-off-by: Chen Jun <chenjun102@huawei.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/powerpc/boot/dts/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/Makefile b/arch/powerpc/boot/dts/Makefile
index fb335d05aae8..e3bc7fb1d17a 100644
--- a/arch/powerpc/boot/dts/Makefile
+++ b/arch/powerpc/boot/dts/Makefile
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0
 
-subdir-y += fsl
+# subdir-y += fsl
 
 dtstree		:= $(srctree)/$(src)
-dtb-$(CONFIG_OF_ALL_DTBS) := $(patsubst $(dtstree)/%.dts,%.dtb, $(wildcard $(dtstree)/*.dts))
+# dtb-$(CONFIG_OF_ALL_DTBS) := $(patsubst $(dtstree)/%.dts,%.dtb, $(wildcard $(dtstree)/*.dts))
-- 
2.26.2

